include/openmc/constants.h

```diff
@@ -69,27 +69,25 @@ constexpr double EXTSRC_REJECT_FRACTION {0.05};
 // ============================================================================
 // MATH AND PHYSICAL CONSTANTS
 
-// Values here are from the Committee on Data for Science and Technology
-// (CODATA) 2014 recommendation (doi:10.1103/RevModPhys.88.035009).
-
-// TODO: cmath::M_PI has 3 more digits precision than the Fortran constant we
-// use so for now we will reuse the Fortran constant until we are OK with
-// modifying test results
-constexpr double PI {3.1415926535898};
+// TODO: replace with <numbers> when we go for C++20
+constexpr double PI {3.141592653589793238462643383279502884L};
 const double SQRT_PI {std::sqrt(PI)};
```

Comment:
Should SQRT_PI also be constexpr? Then it can be used in other constexpr expressions.

Reply:
Not if we're calculating it with `std::sqrt`, which is not `constexpr` per the C++ standard